app/models/manageiq/providers/azure/cloud_manager/event_parser.rb

```diff
@@ -11,10 +11,11 @@ def self.event_to_hash(event, ems_id)
     event_hash = {
       :source     => "AZURE",
       :timestamp  => event["eventTimestamp"],
-      :message    => event["description"].presence,
+      :message    => event["description"].presence || event["operationName"]["localizedValue"].presence,
```

Comment:
What do you think about `event.dig(*%w[operationName localizedValue]).presence`?
And you'll get the free insurance against potential `nil[]` call error: https://devdocs.io/ruby~2.4/hash#method-i-dig

Reply:
I don't think it's possible to get a nil call error if we explicitly select `operationName`, since it's part of the output schema, though I guess it couldn't hurt.

Reply:
Ok, done.